forch/forchestrator.py

```diff
@@ -73,6 +73,9 @@ def __init__(self, config):
 
     def initialize(self):
         """Initialize forchestrator instance"""
+        self._forch_metrics = ForchMetrics(self._config.varz_interface)
+        if self._forch_metrics:
```

Comment:
why is this check necessary?

Reply:
Yup it is not. Removing the checks from forchestrator.

Reply:
Addressed the issue.